.github/workflows/publish_to_pub_workflow.yaml

```
@@ -19,15 +16,15 @@ jobs:
     outputs:
       package_name: ${{ steps.extract_name.outputs.value }}
     steps:
-      - name: Extract package name
+      - name: 📦 Extract package name
         id: extract_name
         shell: bash
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:31: Double quote to prevent globbing and word splitting

(shellcheck)